spec/features/admin_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin' do
+  scenario 'admin can delete user' do
+    admin = create(:user, admin: true)
+    create(:user, email: 'example-user@mail.com')
+    sign_in admin
+
+    visit root_path
+    find(:xpath, "//a[@href='/users/2']").click
```

Comment:
There is a high probability, that this test will fail in the future for some reasons:
- Adding any other link to '/users/2' will make it fail, as there will more than one match - and we use that path for show, so it will probably be there sooner or later
- Users path can change - always use `user_path(user)` instead
- You are assuming the id of the user (2), and small change to this test or to the way records are cleaned between tests can break it

I've just explained it further in the mail.